tests/drivers/stepper/stepper_api/src/main.c

```diff
@@ -140,15 +140,47 @@ ZTEST_F(stepper, test_target_position_w_fixed_step_interval)
 
 	(void)stepper_move_to(fixture->dev, pos);
 
-	/* timeout is set with 20% tolerance */
+	/* timeout is set with 30% tolerance */
 	POLL_AND_CHECK_SIGNAL(stepper_signal, stepper_event, STEPPER_EVENT_STEPS_COMPLETED,
-			      K_MSEC(pos * 120));
+			      K_MSEC(pos * 130));
```

Comment:
Might as well make the 130 a define or a config

Reply:
Would adding Kconfig file with the parameter to the test and then adding the parameter to the testcase.yaml be a good solution?

Reply:
Yeah, that would be great actually. Thanks.